config/install/views.view.alerts_rearrange.yml

```diff
@@ -369,9 +369,9 @@ display:
         - user.permissions
       tags:
         - 'config:field.storage.node.field_alert_place'
-  page_1:
```

Comment:
@mkdok I believe you, but ... what does this change do to fix the menu links? It seems mysterious :)

Reply:
https://github.com/open-y-subprojects/openy_node_alert/pull/123/files#r1344292711 @froboy